src/current/_includes/v23.2/cdc/mux-rangefeed.md

```diff
@@ -25,4 +25,8 @@ Use the following workflow to enable `MuxRangefeed`:
     {% include_cached copy-clipboard.html %}
     ~~~ sql
     RESUME JOB {job ID};
-    ~~~
\ No newline at end of file
+    ~~~
+
+{{site.data.alerts.callout_info}}
+`MuxRangefeed` is enabled by default in v24.1 and later versions. However, if you have running changefeeds created in an earlier version of CockroachDB and upgrade to v24.1+, the `MuxRangefeed` functionality will not apply to these changefeeds. Once you have [upgraded](../v24.1/upgrade-cockroach-version.html) to v24.1, we recommend [pausing]({% link {{ page.version.version }}/pause-job.md %}) and then [resuming]({% link {{ page.version.version }}/resume-job.md %}) existing changefeeds to ensure all changefeeds benefit from the performance improvements of `MuxRangefeed`.
```

Comment:
Is this accurate how MuxRangefeed will not be enabled for existing changefeeds if they were created on an earlier version? I believe pausing/resuming once on a v24.1 cluster would fix that?

Reply:
Is this in a release note? I don't think a restart is necessary.

Reply:
No, this was something @amruss discussed with me. 

@jayshrivastava is there any way to confirm whether that is necessary or not?

Reply:
Looking the code, I think it should for changefeeds without pausing and unpausing them once 24.1 is active.

Reply:
Thank @jayshrivastava! I also checked with Erik and he confirmed. So I have revised the PR so that we're just mentioning that Mux is on by default. Would appreciate it if you could take a look at the revisions!